src/js/player.js

```diff
@@ -389,6 +387,9 @@ class Player extends Component {
 
     this.el_ = this.createEl();
 
+    // Set default value for lastPlaybackRate
+    this.cache_.lastPlaybackRate = this.defaultPlaybackRate();
```

Comment:
Is it appropriate to use `defaultPlaybackRate` here? i.e. will everything be set up properly at this point? Since the element exists, I made the assumption that it would.

Reply:
some techs, namely flash, don't support playbackrate. However, if it isn't supported or the tech isn't ready, it'll just return the default value of 1.0, so, this is probably fine. https://github.com/videojs/video.js/blob/4e79a04579d8b3748ddd4c42707fd5a8bf68cce8/src/js/player.js#L3090-L3099